bin/c.js

```diff
@@ -8,6 +8,7 @@ const { missingCommand } = require('./lib/c');
 // The basic program, which uses sub-commands.
 program
     .command('d <command>', 'Shortcuts to control Docker.')
+    .command('db <command>', 'Shortcuts to help with the database.')
```

Comment:
Hmm, should this be mongo? What if we wanted to do the same for Redis or MySQL or some other database?